src/types.ts

```diff
@@ -74,6 +75,7 @@ type MessageStoreTable = {
   contextId: string | null;
   parentId: string | null;
   permissionGrantId: string | null;
+  prune: string | null;
```

Comment:
Boolean won't work?

Reply:
As always, there's something funky with one of the 3 SQL dialects we use 😆 

In this case it's the driver, maybe we can look for a different one at some point, I haven't done any digging as this was something Adam did before.

https://github.com/TBD54566975/dwn-sql-store/blob/2e354e9141e39275da3bca0a4b0cad628696a7c2/src/utils/sanitize.ts#L44